File: Modul 3/Soal 5/PRAK305-2410817210010-AryaArrozzaRidhoSyaputra.c
#include <stdio.h>

int main()
{
    int s, m, h, d;

    scanf("%d", &s);

    if (s < 86400) {
        m = s / 60;
        h = m / 60;
        s %= 60;
        m %= 60;
        printf("%02d:%02d:%02d", h, m, s);
    }
    else {
        d = s / 86400;
        m = s / 60;
        h = m / 60;
        h %= 24;
        s %= 60;
        m %= 60;
        printf("%d hari %02d:%02d:%02d", d, h, m, s);
    }

    return 0;
}

File: Modul 3/Soal 5/PRAK305-2410817210010-AryaArrozzaRidhoSyaputra.py
s = int(input())

if s < 86400:
    m = s // 60
    h = m // 60
    s %= 60
    m %= 60
    print(f"{h:02}:{m:02}:{s:02}")
else:
    d = s // 86400
    m = s // 60
    h = m // 60
    h %= 24
    s %= 60
    m %= 60
    print(f"{d} hari {h:02}:{m:02}:{s:02}")